### ISSUE TITLE
set_collapsed() interaction with tabbed layouts causes css bug

### ISSUE THREAD

**Issue by sachittandukar** [htmlburger/carbon-fields#274]
Hi,

There is issue with field inside complex field. Setting width to the fields doesn't make any effect. All fields inside complex field always take 100% width.

Here's my code: 
```PHP
Container::make( 'post_meta', 'Showcase Additional Content' )
->where( 'post_type', '=', 'page' )
         ->where( 'post_template', 'IN', [
	         'templates/template-sst-showcase-home1.php',
	         'templates/template-sst-showcase-home2.php',
	         'templates/template-sst-showcase-home3.php'
         ] )
         ->add_tab( __( 'Theme Color Section' ), [
	         Field::make( 'color', 'showcase_theme_color', __( 'Theme Color' ) ),
         ] )
         ->add_tab( __( 'Banner Section' ), [
	         Field::make( 'complex', 'showcase_banner', __( 'Banners List' ) )
	              ->set_layout( 'tabbed-vertical' )
	              ->set_collapsed( true )
	              ->set_max( 5 )
	              ->setup_labels( [
		              'plural_name'   => __( 'Banners' ),
		              'singular_name' => __( 'Banner' ),
	              ] )
	              ->add_fields( [
		              Field::make( 'image', 'image', __( 'Image' ) )->set_required( true )->set_width( 20 ),
		              Field::make( 'rich_text', 'content', __( 'Details' ) )->set_width( 75 ),
		              Field::make( 'text', 'button', __( 'Button Name' ) )->set_width( 50 ),
		              Field::make( 'text', 'url', __( 'Button URL' ) )->set_width( 50 ),
	              ] )
	              ->set_header_template( 'Image Attachment ID: <%- image %>' )
         ] )
```

**Comment by atanas-angelov-dev** [comment]
Hi @sachittandukar ,

Thanks for the report - it seems this is a bug caused by the css class `.collapsed` in combination with tabbed layouts.
If you remove the `set_collapsed( true )` call it will be resolved in your case.

Even if it did not trigger the bug, you should remove `set_collapsed()` anyway since it's only relevant for the default complex layout and does not affect `tabbed-horizontal` and `tabbed-vertical` (there is nothing to collapse in a tabbed layout).

**Comment by atanas-angelov-dev** [comment]
The issue has been fixed in 2.0.2.
